src/nvim/runtime.c

```diff
@@ -269,9 +269,54 @@ void set_context_in_runtime_cmd(expand_T *xp, const char *arg)
   xp->xp_pattern = (char *)arg;
 }
 
-static void source_callback(char *fname, void *cookie)
+/// Source all .vim and .lua files in "fnames" with .vim files given higher priority.
```

Comment:
```suggestion
/// Source all .vim and .lua files in "fnames". If both "x.vim" and "x.lua" are found, the vim file is sourced first.
```

Reply:
This isn't quite true since if `x.vim`, `y.vim`, `x.lua` and `y.lua` are found they are sourced in that order.